client/src/components/Tooltips.tsx

```diff
@@ -46,9 +46,9 @@ export const AdviceTooltipIcon: React.FC<AdviceIconProps> = ({
   icon,
 }) => {
   const toolTip = [
```

Comment:
this should be a fragment (`<>`) and not a list - lists are really only for results of `map`

Reply:
where is a list happening here? is it the ({}) => ?

Reply:
strike that, just replace `[` with `(`